crates/shim/src/cgroup.rs

```diff
@@ -126,12 +152,46 @@ pub fn collect_metrics(pid: u32) -> Result<Metrics> {
     Ok(metrics)
 }
 
+fn set_cpu_usage(cpu_usage: &mut CPUUsage, stat: &String) {
+    let lines: Vec<&str> = stat.lines().collect();
+
+    for line in lines {
+        let parts: Vec<&str> = line.split_whitespace().collect();
+
+        if parts.len() != 2 {
+            continue;
+        }
+
+        match parts[0] {
+            "usage_usec" => {
+                cpu_usage.set_total(parts[1].parse().unwrap());
```

Comment:
How about use `unwrap_or_default` to avoid panic

Reply:
done.